frontend/package.json

```diff
@@ -10,10 +10,14 @@
     "axios": "^0.26.1",
     "bootstrap": "^5.1.3",
     "bootstrap-icons": "^1.8.1",
+    "javascript-time-ago": "^2.5.4",
+    "moment": "^2.29.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-markdown": "^8.0.3",
     "react-router-dom": "^6.2.2",
     "react-scripts": "5.0.0",
+    "react-time-ago": "^7.2.1",
```

Comment:
Hmm, 2 new packages for time ago sounds a bit overkill. But it seems to work now, so I dont mind much.